models.py

```diff
@@ -614,10 +668,11 @@ def run_developer(target_units, lid, forms, buildings, supply_fname,
     if new_buildings is None or len(new_buildings) == 0:
         return
 
+    register_btype_distributions(buildings.to_frame(["building_type_id"]))
```

Comment:
If this is running on all of buildings, than why is it in the `for lid, _ in parcels.large_area_id` loop? Can it be called once before that loop?

Also the pr mentioned that it took into account "proportions ... by large area", but I don't see that in the code. Did you mean to run this in the loop on `la_orig_buildings`?

Reply:
You're right - I didn't catch that `la_orig_buildings` isn't getting passed in to `run_developer()`. 

What would you prefer?
- The `register_btype_distributions` function gets called on the full region-wide building table once, and we move this function one level higher, or
- The `register_btype_distributions` function gets called within each large area building table, and we modify arguments to take in a large-area-only buildings table?

Reply:
Can we call it in the loop but before the function with `la_orig_buildings`?

Reply:
Yes - testing this now. I'll add a new commit when ready.

Reply:
Ok, thanks!